.eslintrc

```diff
@@ -90,10 +90,10 @@
     "no-use-before-define": 2,
     "radix": 2,
     "curly": 2,
-    "no-fallthrough": 0,
-    "default-case": 0,
-    "no-plusplus": 0,
-    "dot-notation": 0
+    "no-fallthrough": 2,
+    "default-case": 2,
+    "no-plusplus": 2,
```

Comment:
I think ++ should be kept as it is the idiomatic increment operator. += 1 is very non-idiomatic.